MatrixSDK/Crypto/KeyBackup/MXKeyBackup.m

```diff
@@ -44,7 +44,10 @@
 
 @interface MXKeyBackup ()
 {
-    MXSession *mxSession;
+    MXCrypto *crypto;
```

Comment:
Use a weak reference here to avoid retain cycle with `MXCrypto.keybackup`. Currently MXKeyBackup as a strong reference to MXCrypto and MXCrypto has a strong reference to MXKeyBackup.